schema/soldier-schema.js

```diff
@@ -0,0 +1,18 @@
+const soldier = {
+  body: {
```

Comment:
the body is `soldier`-like.. soldier doesn't have a body (well, I meant in the schema.. 😄 )